api-playground/asyncapi/setup.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "AsyncAPI Setup"
+description: "Reference AsyncAPI endpoints in your docs pages"
```

Comment:
are they technically endpoints?

Reply:
or something else

Reply:
how about "Create reference pages for websockets with AsyncAPI"

Reply:
lets do it